pkg/api/handlers/compat/images.go

```diff
@@ -409,14 +409,20 @@ func GetImages(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Failed get images"))
 		return
 	}
-	var summaries = make([]*entities.ImageSummary, len(images))
-	for j, img := range images {
+
+	summaries := []*entities.ImageSummary{} // nolint
```

Comment:
Should we still preallocate the memory with `make([]*entities.ImageSummary, 0, len(images))`?

Reply:
Good idea.